src/plugins/ui_stateful/clipboard.ts

```diff
@@ -552,7 +552,9 @@ export class ClipboardPlugin extends UIPlugin {
     }
```

Comment:
missing task number in commit message

Reply:
And we try to follow odoo guidelines and to keep the lines of the commit messages within 70(?) characters